### PR TITLE
silabs-multiprotocol: Add amd64 support

### PR DESCRIPTION
**Pull request by agners**
Requires: https://github.com/home-assistant/builder/pull/160

### CODE REVIEW

File: silabs-multiprotocol/rootfs.amd64/usr/local/bin/zigbeed [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "Starting zigbeed with aarch64 emulation..."
```
Comment (pvizeli) [inline silabs-multiprotocol/rootfs.amd64/usr/local/bin/zigbeed:3]: please use bashio

Comment (pvizeli) [reply silabs-multiprotocol/rootfs.amd64/usr/local/bin/zigbeed:3]: I guess you will only use it as wrapper? Mabe have it then only as comment?

Comment (agners) [reply silabs-multiprotocol/rootfs.amd64/usr/local/bin/zigbeed:3]: Yeah its only mean to be a wrapper. I make it a comment.

**Comment by agners** [comment]
This will also fix #2741.

**Review by pvizeli — APPROVED** [review]

**Pull request merged**